java-checks/src/test/files/checks/spring/SpringScanDefaultPackageCheckInAPackage.java

```diff
@@ -0,0 +1,16 @@
+package files.checks.spring;
+
+import java.util.Map;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.ServletComponentScan;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan
+@SpringBootApplication
+@ServletComponentScan
+
+@ComponentScan(basePackages = "") // Noncompliant
+@ComponentScan(basePackageClasses = App.class)
+
```

Comment:
nitpick: also add `@SpringBootApplication(scanBasePackages = "") // Noncompliant` and `@ServletComponentScan("") // Noncompliant`